src/test/java/com/incognia/api/IncogniaAPITest.java

```diff
@@ -101,6 +102,9 @@ void testRegisterSignup_whenDataIsValid() {
     expectedEvidence.put("device_integrity", deviceIntegrity);
 
     assertThat(signupAssessment.getEvidence()).containsExactlyInAnyOrderEntriesOf(expectedEvidence);
+
+    Reason expectedReason = Reason.builder().code("trusted_location").source("local").build();
```

Comment:
What about using the constants rather than a string?

Reply:
done